src/box/sql/vdbe.c

```diff
@@ -3213,9 +3162,9 @@ case OP_Last: {        /* jump */
 			goto abort_due_to_error;
 		pC->nullRow = (u8)res;
 		pC->cacheStatus = CACHE_STALE;
-		if (pOp->p2>0) {
-			VdbeBranchTaken(res!=0,2);
-			if (res) goto jump_to_p2;
+		if (pOp->p2 > 0) {
+			if (res != 0)
+				goto jump_to_p2;
```

Comment:
It might be easier to do `if (pOp->p2 > 0 && res != 0) ...` instead of 2 ifs. Less indentation.

Reply:
Thank you! Fixed.